tests/keras_tests/feature_networks_tests/feature_networks/layer_fusing_test.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2021 Sony Semiconductors Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from abc import ABC
+import tensorflow as tf
+from model_compression_toolkit.tpc_models.default_tp_model import get_op_quantization_configs
+import model_compression_toolkit as mct
+from tests.keras_tests.feature_networks_tests.base_keras_feature_test import BaseKerasFeatureNetworkTest
+if tf.__version__ < "2.6":
+    from tensorflow.keras.layers import Conv2D, DepthwiseConv2D, Dense, Activation, ReLU, Add
+else:
+    from keras.layers import Conv2D, DepthwiseConv2D, Dense, Activation, ReLU, Add
+
```

Comment:
We have no test here for LayerFilterParams cases (where a specific layer type should be fused). Please test some of these patterns as well.

Reply:
added LayerFilterParams  to tests